### ISSUE TITLE
Downloading external resources is disabled without quickfix or search term in preferences for XML

### ISSUE THREAD

**Issue by vogella** [eclipse-lemminx/lemminx#1709]
Currently one XML file taken from an TME4 example shows lots of errors. 

![downloading-external-resources](https://github.com/user-attachments/assets/efbe810f-a9f4-4c0b-a30b-90e88eb536d9)

Search preference does not reveal anything:

![image](https://github.com/user-attachments/assets/44993e8a-9e7d-447e-b0e6-58704562a1b3)

![image](https://github.com/user-attachments/assets/4914b80f-ba73-44f6-a612-973c11cbd918)

After a while I found https://github.com/eclipse-wildwebdeveloper/wildwebdeveloper/issues/1289 which lead to me to preference setting in XML from WWD / Lemmix.

I suggest to add at least "download", "downloading", "external" to the preference search term.
In addition a perfect solution would be to have a quickfix for this change, changing the preference from the editor directly.

**Comment by mickaelistria** [comment]
All good ideas, PRs welcome ;)

**Comment by vogella** [comment]
This repo seems to be "only" the language server, right? So where are the preference pages implemented? WWD?

**Comment by mickaelistria** [comment]
Yes, this would be in Wild Web Developer. I'm closing the issue here.

**Comment by angelozerr** [comment]
@vogella you should have those preferences:

![image](https://github.com/user-attachments/assets/11e2fdc4-055e-4b51-a46e-2dd6b056de32)

**Comment by angelozerr** [comment]
> In addition a perfect solution would be to have a quickfix for this change, changing the preference from the editor directly.

You should have a quick fix `Force download of ...` which should download the DTD, XSD that you ar eassociating (it is working on vscode-xml)

**Comment by vogella** [comment]
@angelozerr I have the preferences but the search terms have not been maintained and I do NOT have a quick-fix in Eclipse. Maybe this is implemented outside of the language-server in vscode or something is broken in Eclipse.

**Comment by angelozerr** [comment]
The quick fix is managed by the language server. I don't know why it doesn't work on WWD (I'm sorry I have no time to investigate the problem).

**Comment by mickaelistria** [comment]
Reopening as LS might be involved according to last comment
